### DISCUSSION TITLE
Book Order?

### DISCUSSION THREAD

**Discussion by ericpyle** [bible-technology/scripture-burrito#283] [Archive]
I recently discovered the removal of canon specifications:
https://github.com/bible-technology/scripture-burrito/discussions/277

- If SB is to be used for the DBL in the future (e.g. uploading from Paratext), there is at least one use case (download as epub) for book order information. Chronological Bibles (sign language video) are also highly ordered publications that are non-standard orderings. Beyond that, I just wonder if SB does not include book ordering, what might that look like in terms of publishing partners who may need that information? Any ideas where they should get that, and who is responsible for doing it? If it should be a custom extension of SB, can someone provide an example of what that would look like? Or should it be a separate specification altogether?

- Paratext Publishing Assistant (PA) also uses that information for creating printed Bible and print on demand files. 

- At some point I was aware that Paratext and (possibly Paratext Registry) were planning to have a translator-friendly way for translators to specify how they'd like to see their books published (including order) and that this would be a part of their translation milestones.

**Comment by jonathanrobie** [comment]
The same content can be published in multiple orders, e.g. for an Orthodox audience or a Protestant audience.   I'm OK with adding this back, but only if we do a redesign.  In particular, I think we need to support:

1. Standard named orders
2. The ability to support multiple orders in a single project
3. Custom orders, e.g. for chronological bibles

For standard named orders, though, there may be little value to saying "this can be published in the following orders" - that's implicit.  Who would stop you from doing so?

**Comment by jonathanrobie** [comment]
Versification could be extended to support:

- standard specifications
- a format for custom orders

**Comment by jag3773** [comment]
Possibly consider:

- Making ingredients an ordered list (Tim is opposed)
- Provide an example of transferring app specific files in a burrito
- Provide a role for a canon specification file

**Reply by jonathanrobie** [reply]
I would also be opposed to making ingredients an ordered list.  Especially since some publishers need to publish more than one order from the same source text, using an ordered list doesn't solve the basic problem here.  We want to keep project order and publication order loosely coupled.  An extension for publishers, in the extensibility section, seems like the best solution here, especially if publishers can get together and agree on a common set of extensions.  I'd like to invite at least PTXPrint, Scripture App Builder, and Publishing Assistant to work together on this.

**Comment by jonathanrobie** [comment]
We may want order in published DBL artifacts, but not decide we do not want order in project burritos.
